source/docs/api-reference/device-specific/talonfx/closed-loop-requests.rst

```diff
@@ -306,3 +306,17 @@ Once the gains are configured, the Motion Magic® request can be sent to the Tal
 
          // set position to 10 rotations
          m_talonFX.SetControl(request.WithPosition(10_tr));
+
+Continuous Mechanism Wrap
+-------------------------
+
+A continuous mechanism is a mechanism with unlimited travel in any direction, and whose rotational position can be represented with multiple unique position values. Some examples of continuous mechanisms are swerve drive steer mechanisms or turrets (without cable management).
+
+``ContinuousWrap`` (`Java <https://api.ctr-electronics.com/phoenix6/release/java/com/ctre/phoenix6/configs/ClosedLoopGeneralConfigs.html#ContinuousWrap>`__, `C++ <https://api.ctr-electronics.com/phoenix6/release/cpp/classctre_1_1phoenix6_1_1configs_1_1_closed_loop_general_configs.html#a10ee9d992c59de7cb649c2001f2c4c8f>`__) is a mode of closed loop operation that enables the Talon to take the "shortest path" to a target position for a continuous mechanism. It does this by assuming that the mechanism is continuous within 1 rotation.
+
+For example, if a Talon is currently at 2.1 rotations, it knows this is equivalent to 3.1 rotations, 1.1 rotations, 0.1 rotations, -0.9 rotations, etc. If that talon is then commanded to a position of 0.8 rotations, instead of driving backwards 1.3 rotations or forwards 0.7 rotations, it will drive backwards 0.3 rotations to a target of 1.8 rotations.
```

Comment:
This sentence is somewhat confusing. Can a picture be added that highlights wrapping behavior?